### PR TITLE
Small features

### LINKED ISSUE #112: Make Shift+V have paradropping units delay parachute opening until e.g. 150m ATL
I really love the paradrop feature that one can trigger by pressing V on an aircraft. Together with setFlyInHeight it makes for a great simulation of a static line parachute drop (instant parachute deployment). 

However, sometimes I would like to have the AI simulate a High Altitude Low Opening (HALO) jump, e.g. in situations where enemy fire would pick them all off in the sky with a "traditional" jump.
Therefore I'd like to suggest extending the feature by adding a second keybinding that will have them delay their parachute opening until X meters ATL (where X is some number around 200-ish, might need some testing so they don't kill or injure themselves ^^)

### LINKED ISSUE #133: Artillery module: New spread range option
Implementation: Random target points are selected within the given radius.

### PR DESCRIPTION
**Pull request by oOKexOo**
**When merged this pull request will:**
- Resolves #133.
- Resolves #112.

### CODE REVIEW

File: @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 #include "\achilles\modules_f_ares\module_header.hpp"
 
-private ["_objects","_guns","_rounds","_ammo","_targetPos", "_artilleryAmmoDisplayName", "_ammoSelectedDisplayName"];
+private ["_objects","_guns","_rounds","_ammo","_targetPos", "_artilleryAmmoDisplayName", "_ammoSelectedDisplayName", "_precision"];
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf]: Why private

File: @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf [hunk 2]
```diff
@@ -102,6 +103,7 @@ if (_mode == 0) then
 	_rounds = parseNumber (_pickFireMissionResult select 1);
 	_ammo = (_artilleryAmmo select (_pickFireMissionResult select 2));
 	_ammoSelectedDisplayName = (_artilleryAmmoDisplayName select (_pickFireMissionResult select 2));
+	_precision = parseNumber (_pickFireMissionResult select 4);
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf]: params

File: @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_chute.sqf [hunk 3]
```diff
@@ -28,19 +28,29 @@ if (!isPlayer _unit) then
...
 
-	// if the unit already has a chute
-	if (backpack _unit != "" and {getText (configfile >> "CfgVehicles" >> backpack _unit >> "backpackSimulation") isEqualTo "ParachuteSteerable"}) then
+	private _backpackClass = backpack _unit;
+	if (_backpackClass != "") then
 	{
+		private _container = backpackContainer _unit;
+		private _weapon_cargo = getWeaponCargo _container;
+		private _magazine_cargo = getMagazineCargo _container;
+		private _item_cargo = getItemCargo _container;
```
Comment (CreepPork) [inline @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_chute.sqf]: Add some spaces, so the code isn't so cramped and more easily read

Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/functions_f_achilles/functions/features/fn_chute.sqf]: done

File: @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf (hunk 2 above)
Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf]: not possible here

File: @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf (hunk 1 above)
Comment (oOKexOo) [reply @AresModAchillesExpansion/addons/modules_f_ares/FireSupport/functions/fn_FireSupportArtilleryFireMission.sqf]: Required for scoping

**Pull request merged**